### PR TITLE
Added docker health check info

### PR DESCRIPTION
**Pull request by joel-hamill**
## What are your changes?
- Also cleaned up "healthcheck" terminology.
## What type of changes does your doc introduce?
- [ ] Bug fix
- [ ] New feature 

## What is the urgency level of this change?
- [ ] Blocker
- [ ] High
- [ ] Medium

## I have completed these items:
- [ ] I have tested all commands and code snippets.
- [ ] I have built locally and tested for broken links and formatting.
- [ ] I have made this change for all affected versions (e.g. 1.7, 1.8, and 1.9).
- [ ] My doc follows the style guide: https://github.com/dcos/dcos-docs#contributing.

### CODE REVIEW

File: 1.7/usage/service-discovery/load-balancing.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ When you launch a set of tasks when these labels, we distribute them to all of t
...
+We also recommend taking advantage of Mesos health checks. Mesos health checks are surfaced to the load balancing layer. **Marathon** only converts **command** health checks to Mesos health checks. You can simulate HTTP health checks via a command similar to `test "$(curl -4 -w '%{http_code}' -s http://localhost:${PORT0}/|cut -f1 -d" ")" == 200`. This ensures the HTTP status code returned is 200. It also assumes your application binds to localhost. The `${PORT0}` is set as a variable by Marathon. We do not recommend using TCP health checks as they can be misleading as to the liveness of a service.
```
Comment (sargun) [inline 1.7/usage/service-discovery/load-balancing.md]: @aquamatthias Is this still true, or is it fixed?

File: 1.8/usage/service-discovery/load-balancing-vips/index.md [hunk 2]
```diff
@@ -23,8 +23,16 @@ When you launch a set of tasks with these labels, DC/OS distributes them to all
...
 
-#### Healthchecks
-We also recommend taking advantage of Mesos healthchecks. Mesos healthchecks are surfaced to the load balancing layer. **Marathon** only converts **command** healthchecks to Mesos healthchecks. You can simulate HTTP healthchecks via a command similar to `test "$(curl -4 -w '%{http_code}' -s http://localhost:${PORT0}/|cut -f1 -d" ")" == 200`. This ensures the HTTP status code returned is 200. It also assumes your application binds to localhost. The `${PORT0}` is set as a variable by Marathon. We do not recommend using TCP healthchecks as they can be misleading as to the liveness of a service.
+#### Health checks
+We also recommend taking advantage of Mesos health checks. Mesos health checks are surfaced to the load balancing layer. **Marathon** only converts **command** health checks to Mesos health checks. You can simulate HTTP health checks via a command similar to:
```
Comment (sargun) [inline 1.8/usage/service-discovery/load-balancing-vips/index.md]: Isn't this redundant, based on the earlier stuff in load-balancing.md?

File: 1.9/usage/service-discovery/mesos-dns/custom-domain-service-discovery.md [hunk 3]
```diff
@@ -146,7 +146,7 @@ DC/OS uses Mesos-DNS for internal service discovery. While the `.mesos` domain c
           </td>
 
           <td class="confluenceTd">
-            A specific agent to pin Mesos-DNS to. Since Mesos-DNS will be acting as a delegate, its IP must remain static. For multiple Mesos-DNS instances (and redundancy), the LIKE constraint operator can be used to define hosts using regex. See <a href="https://mesosphere.github.io/marathon/docs/constraints.html" class="external-link" rel="nofollow">https://mesosphere.github.io/marathon/docs/constraints.html</a> for more information. As a best-practice, we also recommend using healthchecks to ensure that Mesos-DNS remains running.
+            A specific agent to pin Mesos-DNS to. Since Mesos-DNS will be acting as a delegate, its IP must remain static. For multiple Mesos-DNS instances (and redundancy), the LIKE constraint operator can be used to define hosts using regex. See <a href="https://mesosphere.github.io/marathon/docs/constraints.html" class="external-link" rel="nofollow">https://mesosphere.github.io/marathon/docs/constraints.html</a> for more information. As a best-practice, we also recommend using health checks to ensure that Mesos-DNS remains running.
```
Comment (sargun) [inline 1.9/usage/service-discovery/mesos-dns/custom-domain-service-discovery.md]: Didn't we get rid of this with https://gist.github.com/sargun/7137a3a04a9678a8e7bfd8c3e79199ef?

File: 1.7/usage/service-discovery/load-balancing.md (hunk 1 above)
Comment (aquamatthias) [inline 1.7/usage/service-discovery/load-balancing.md]: This is true for DC/OS 1.8. Mesos has added support for http/tcp health check in 1.1.

Comment (joel-hamill) [inline 1.7/usage/service-discovery/load-balancing.md]: @aquamatthias is this true for 1.7?

File: 1.8/usage/service-discovery/load-balancing-vips/index.md (hunk 2 above)
Comment (joel-hamill) [inline 1.8/usage/service-discovery/load-balancing-vips/index.md]: `load-balancing.md` is a topic that only exists in 1.7. This is the 1.8 version of that topic ^^.

File: 1.9/usage/service-discovery/mesos-dns/custom-domain-service-discovery.md (hunk 3 above)
Comment (joel-hamill) [inline 1.9/usage/service-discovery/mesos-dns/custom-domain-service-discovery.md]: yes, more specifically with: https://github.com/dcos/dcos-docs/pull/730. see the merge conflict below. :)

File: 1.7/usage/service-discovery/load-balancing.md (hunk 1 above)
Comment (joel-hamill) [inline 1.7/usage/service-discovery/load-balancing.md]: @aquamatthias ^^

**Pull request merged**